extensions/sql-bindings/package.json

```diff
@@ -56,6 +56,13 @@
   },
   "devDependencies": {
     "@types/fs-extra": "^5.0.0",
-    "tslint": "^5.8.0"
+    "tslint": "^5.8.0",
+    "should": "^13.2.1",
```

Comment:
I'm not seeing this be used in the tests - make sure you go through and verify each of these is being used.

Reply:
This is used in addSQLBindingQuickPick.test.ts

Reply:
Ah oops, must have had that collapsed in Github when I searched